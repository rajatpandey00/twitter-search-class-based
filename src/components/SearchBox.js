import React, { useState } from 'react';
import { Button } from 'react-bootstrap'

// @flow
const SearchBox = (props) => {
    return (
            <div className="search-box">
                <input
                    placeholder="Search by User name..."
                    value={props.query}
                    onChange={props.handleInputChange}
                />
                <Button onClick={props.getData}>Submit</Button>
            </div>
    );
}

export default SearchBox;