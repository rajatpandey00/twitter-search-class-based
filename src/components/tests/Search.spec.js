import React from 'react';
import renderer from 'react-test-renderer';

import SearchBox from '../SearchBox';

describe('SearchBox test', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<SearchBox />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});