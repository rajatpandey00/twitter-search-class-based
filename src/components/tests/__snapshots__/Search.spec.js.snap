// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox test snapshot renders 1`] = `
<div>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <input
      placeholder="Search by User name..."
      style={
        Object {
          "margin": "10px",
        }
      }
    />
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Submit
    </button>
  </div>
</div>
`;
